code/modules/crew_objectives/medical_objectives.dm

```diff
@@ -2,7 +2,7 @@
 
 /datum/objective/crew/morgue //Ported from old Hippie
 	explanation_text = "Ensure the Medbay has been cleaned of any corpses when the shift ends."
-	jobs = "chiefmedicalofficer,geneticist,medicaldoctor"
+	jobs = "chiefmedicalofficer,geneticist,medicaldoctor,paramedic,"
```

Comment:
🤔

Reply:
was iffy about this and didn't mean to push it